lightly/data/_video.py

```diff
@@ -11,22 +11,22 @@
 
 
 def _video_loader(path, timestamp, pts_unit='sec'):
-    """Reads a frame from a video at a random timestamp.
+    """Reads a frame from a video at a specific timestamp.
 
     Args:
         path:
             Path to the video file.
         timestamp:
             The timestamp at which to retrieve the frame in seconds.
-        pts_unit:
-            Unit of the timestamp.
+
+    Returns:
+        A PIL image
 
     """
     # random access read from video (slow)
     frame, _, _ = read_video(path,
                              start_pts=timestamp,
-                             end_pts=timestamp,
-                             pts_unit=pts_unit)
+                             end_pts=timestamp)
     # read_video returns tensor of shape 1 x W x H x C
```

Comment:
Why did you remove the `pts_unit`? This can lead to errors because we read the timestamps in seconds.

Reply:
Good catch. I saw that the pts_unit isn't set for the private method, so I thought I would remove it. I forgot to switch to pts_unit='sec'.